src/DN404.sol

```diff
@@ -129,6 +129,11 @@ abstract contract DN404 {
     /// [Etherscan](https://etherscan.io/address/0x000000000022D473030F116dDEE9F6B43aC78BA3)
     address internal constant _PERMIT2 = 0x000000000022D473030F116dDEE9F6B43aC78BA3;
 
+    /// @dev The ZKsync Permit2 deployment.
+    /// If deploying on ZKsync or Abstract, override `_isPermit2(address)` to check against this too.
+    /// [Etherscan](https://era.zksync.network/address/0x0000000000225e31D15943971F47aD3022F714Fa)
+    address internal constant _ZKSYNC_PERMIT_2 = 0x0000000000225e31D15943971F47aD3022F714Fa;
```

Comment:
The abstract permit2 address is actually: 0x6b4420f66De496D18A6c36367cf7f1440fd9289f

Do you need it specifically at the address above?

Reply:
Oh my. fragmentation. Best to get it at the same address!

Reply:
It’s not yet up, https://explorer.testnet.abs.xyz/address/0x6b4420f66De496D18A6c36367cf7f1440fd9289f

so better fix now b4 mainnet.

Reply:
I'll try to get it deployed to the same address